src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -343,22 +354,32 @@ export class AccessibleView extends Disposable {
 		if (provider.id === AccessibleViewProviderId.Chat) {
 			this._register(this._codeBlockContextProviderService.registerProvider({ getCodeBlockContext: () => this.getCodeBlockContext() }, 'accessibleView'));
 		}
+		if (!('verbositySettingsKey' in provider)) {
+			this._storageService.store(`ACCESSIBLE_VIEW_SHOWN_${provider.id}`, true, StorageScope.APPLICATION, StorageTarget.USER);
```

Comment:
I see the `ACCESSIBLE_VIEW_SHOWN_` prefix used in several places. It would be nice to extract this into a constant.